api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -447,6 +453,10 @@ message Admin {
   // Indicates whether :ref:`global_downstream_max_connections <config_overload_manager_limiting_connections>`
   // should apply to the admin interface or not.
   bool ignore_global_conn_limit = 6;
+
+  // Optional routes list for admin which are accessible
+  // if not specified, the allow list check will be skippped
+  AllowListedRoutesArray allow_listed_routes = 7;
```

Comment:
can we just inline the repeated string of routes?  and if the list is empty, that means the allow list check is skipped?